CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # CHANGELOG
 
+## 3.2.0rc0 (2025-07-29)
+
+MLflow 3.2.0rc0 includes several major features and improvements
+
+### Major New Features
+
+- 🧭 **Tracing TypeScript SDK**: MLflow Tracing now supports the TypeScript SDK, making it easier to trace LLM workflows in TypeScript environments.(#16871, @B-Step62)
+- 🧪 **Assessment Backend**: MLflow OSS now supports tracking assessments, enabling structured evaluation of LLM workflows. (#16743, @BenWilson2)
+- 🖥️ **MLflow UI Improvements**: The MLflow UI now features **a redesigned experiment home view** and includes enhancements like pagination on the model page for better usability. (#16464, @frontsideair, #15801, @Flametaa)
+- 📊 **MLflow Telemetry**: Starting with version 3.2.0, MLflow will begin collecting anonymized usage data about how core features of the platform are used. This data contains **no sensitive or personally identifiable information**, and users can opt out of data collection at any time. Check [MLflow documentation](https://mlflow.org/docs/latest/community/usage-tracking) for more details. (#16439, @serena-ruan)
```

Comment:
Telemetry is not a feature. Can we add a separate section for this?

Reply:
ok

Reply:
Shall we create an issue or something to gather any potential feedback on telemetry?